### ISSUE TITLE
tree -s --human not working anymore

### ISSUE THREAD

**Issue by tecepeipe** [rclone/rclone#5868]
The tree command showing size in human readable format isn't working anymore in both 1.57 or latest beta:

rclone tree gdrive-bundle:"Fonts"  -s --human
[    9760619]  /
├── [      38432]  AAHAMASN.TTF
├── [      47460]  ARTISTIK.TTF
├── [      45240]  ASHLYINL.TTF
├── [      55892]  ActionIsShaded.ttf
├── [      87052]  Amandine_Font_by_runaways_rocks.ttf

Tried this on windows 10 64bits against my google drive

I was expecting to see this:

 rclone tree gdrive-bundle:"Fonts"  -s --human
[9.3M]  /
├── [ 38K]  AAHAMASN.TTF
├── [ 46K]  ARTISTIK.TTF
├── [ 44K]  ASHLYINL.TTF
├── [ 55K]  ActionIsShaded.ttf
├── [ 85K]  Amandine_Font_by_runaways_rocks.ttf

But I got that on  1st output  instead, with no K, M, G:

**Comment by tecepeipe** [comment]
actually the feature is there, the new flag is --human-readable, we just need to update the help table (or add both for retrocompatibility)

**Comment by animosity22** [comment]
Feel free to submit a PR to clean it up.

**Comment by albertony** [comment]
> actually the feature is there, the new flag is --human-readable

Yes. This command had a dedicated flag that was removed when we introduced a global flag `--human-readable` (which works with other commands as well). I guess I just forgot to remove this line, which I think is what you refer to?

https://github.com/rclone/rclone/blob/408d9f3e7aa8a88dc36f7c69046f80cc946fc74d/cmd/tree/tree.go#L46

@tecepeipe If you would like to submit a PR for this that would be great? (If not I'll put it on my backlog).
